Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -173,13 +174,20 @@ WarpX::MoveWindow (const int step, bool move_j)
             if (move_j) {
                 shiftMF(*current_fp[lev][dim], geom[lev], num_shift, dir);
             }
-            if (do_pml && pml[lev]->ok()) {
+            if (pml[lev] && pml[lev]->ok()) {
                 const std::array<MultiFab*, 3>& pml_B = pml[lev]->GetB_fp();
                 const std::array<MultiFab*, 3>& pml_E = pml[lev]->GetE_fp();
                 shiftMF(*pml_B[dim], geom[lev], num_shift, dir);
                 shiftMF(*pml_E[dim], geom[lev], num_shift, dir);
             }
-
+#ifdef WARPX_DIM_RZ
+            if (pml_rz[lev] && dim < 2) {
+                const std::array<MultiFab*, 2>& pml_rz_B = pml_rz[lev]->GetB_fp();
+                const std::array<MultiFab*, 2>& pml_rz_E = pml_rz[lev]->GetE_fp();
```

Comment:
```suggestion
                const std::array<amrex::MultiFab*, 2>& pml_rz_B = pml_rz[lev]->GetB_fp();
                const std::array<amrex::MultiFab*, 2>& pml_rz_E = pml_rz[lev]->GetE_fp();
```

Reply:
Fixed. Perhaps a separate cleanup PR can be submitted to add the `amrex` to all of the other unadorned `MultiFabs` in this file.

Reply:
Thanks, same, I can open a cleanup PR to do this.

Reply:
Done in #2579.